code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -21,13 +21,25 @@
 	family_heirlooms = list(/obj/item/toy/cattoy)
 	examine_limb_id = SPECIES_HUMAN
 
-// Prevents felinids from taking toxin damage from carpotoxin
+// Prevents felinids from taking toxin damage from carpotoxin and makes them throw up because of chocolate.
 /datum/species/human/felinid/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H, seconds_per_tick, times_fired)
```

Comment:
species inherent? this sucks. why not stomach inherent instead? genetic trait applied to felinids that modifies their stomach to do it? but i hate species dependent interactions over organ and mutation ones